rpc_jobs/rpc_aio.yml

```diff
@@ -79,6 +79,15 @@
                     default_volume_type: ceph
             cinder_service_backup_driver: cinder.backup.drivers.ceph
             tempest_service_available_swift: false
+      - ironic-multihypervisor:
```

Comment:
How do you test multi-hypervisor in an AIO job? I think we only one hyp not in a container. Is it that multiple virtual ironic nodes are created for testing?

Reply:
Multi-hypervisor refers to being able to create baremetal nodes and virtual nodes. The other scenario for ironic will be "standalone," which won't deploy nova compute so that only ironic builds are supported.